routes/api/v1/users.js

```diff
@@ -1,11 +1,28 @@
 var express = require('express');
 var router = express.Router();
-
 var user = require('../../../models').User
+var bcrypt = require('bcrypt')
+const saltRounds = 10
+const uuidv1 = require('uuid/v1');
+
+router.post("/", function(req, res, next){
+  res.setHeader("Content-type", "application/json")
 
-/* GET users listing. */
-router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
-});
+  if (req.body.password === req.body.passwordConfirmation){
+    user.create({
+      email: req.body.email,
+      password: bcrypt.hashSync(req.body.password, saltRounds),
+      apiKey: uuidv1()
+    })
+    .then(user => {
+    res.status(201).send(JSON.stringify({ apiKey: user.apiKey }))
+    })
+    .catch(error => {
+      res.status(500).send(JSON.stringify("Your information is not correct!"));
+    })
+  } else {
```

Comment:
I would love to see this extrapolated a bit more into separate else cases--like if a password doesn't match or if a field is missing! Might be helpful for future devs working on your code base!!

Reply:
I plan to do this when I get a little further into it. I'll keep you posted!